pom.xml

```diff
@@ -57,21 +60,31 @@
     <sharing_policies.yaml.file>${project.basedir}/src/main/resources/swagger.api/sharing_policies.yaml</sharing_policies.yaml.file>
     <sharing_roles.yaml.file>${project.basedir}/src/main/resources/swagger.api/sharing_roles.yaml</sharing_roles.yaml.file>
 
+    <!--Folio dependencies properties-->
     <folio-module-descriptor-validator.version>1.0.1</folio-module-descriptor-validator.version>
+    <folio-spring-base.version>9.0.0</folio-spring-base.version>
+    <folio-service-tools.version>5.0.0</folio-service-tools.version>
+    <applications-poc-tools.version>2.3.0-SNAPSHOT</applications-poc-tools.version>
```

Comment:
2.4.0-SNAPSHOT is available, hope it will be released soon and we will change to 2.4.0 during our release
https://github.com/folio-org/applications-poc-tools/blob/master/pom.xml#L15